src/utils/patentDataUtils.js

```diff
@@ -0,0 +1,103 @@
+import { API } from 'aws-amplify';
+
+function getPatentData(proteinId){
+    /**
+     * Fetch patent details from the patents database for the given $proteinId
+     */
+    const apiName = 'patentsAPI';
+    const path = '/patents'; 

```

Comment:
should we have a paths/patentsAPI object stored somewhere else for easy edits?

Reply:
are you thinking of a constants file for all the API paths? (I could take care of that as part of https://patent-visualization.atlassian.net/browse/PSV-69 where I'll need to get more APIs working)